resolwe_bio/utils/test.py

```diff
@@ -90,3 +91,12 @@ def prepare_expression(self, f_rc='exp_1_rc.tab.gz',
         """Prepare expression."""
         inputs = {'rc': f_rc, 'exp': f_exp, 'exp_type': f_type, 'exp_name': name, 'source': source}
         return self.run_process('upload-expression', inputs)
+
+class KBBioProcessTestCase(BioProcessTestCase):
+    """Test class for bioinformatics processes that use knowledge base."""
+
+    def setUp(self):
+        """Initialize test files path."""
```

Comment:
Change "Initialize test files path." to the description on what the the management commands do.